### DISCUSSION TITLE
Is it possible to train spectrogram images using MONAI?

### DISCUSSION THREAD

**Discussion by aalfi** [Project-MONAI/MONAI#5848] [Q&A] [answered]
Hi... I am new to deep learning.
I am trying to train spectrogram images (png) using MONAI.
Is it possible to train spec images by changing the transform parameters?

My apologies if this is a dumb question

Thanks in advance!

**Comment by KumoLiu** [comment, marked as answer]
Hi @aalfi, I'm not very familiar with spectrogram images, but I think you can use the MONAI pipeline to conduct a preliminary experiment first. You can find many tutorials in the [tutorial repo](https://github.com/Project-MONAI/tutorials), depending on your task you can choose a suitable tutorial as a start. As a suggestion, you can first comment all augmentation transforms as a pre-experiment and then add the appropriate transforms as needed.

Hope it can help you, thanks!

**Reply by aalfi** [reply]
Thank you @KumoLiu!!
Sorry for the late reply.
I will try to change the transform parameters as you mentioned and let you know the progress.

Thanks again!
